bigbluebutton-html5/imports/ui/components/audio/audio-controls/input-stream-live-selector/component.jsx

```diff
@@ -316,7 +316,7 @@ class InputStreamLiveSelector extends Component {
         aria-label={intl.formatMessage(intlMessages.leaveAudio)}
         label={intl.formatMessage(intlMessages.leaveAudio)}
         accessKey={shortcuts.leaveaudio}
-        data-test="leaveAudio"
+        data-test="leaveListenOnly"
```

Comment:
There's a pretty big inconsistency in the type of quotes we use in this file. Not a big deal, obviously, but since you are modifying these selectors, let's switch to single quotes as well.